src/lib/arrays.js

```diff
@@ -0,0 +1,100 @@
+const getNthElement = (index, array) => {
+  index %= array.length;
+  return array[index];
```

Comment:
Did you notice 'Assignment to function parameter' eslint error in your IDE? This is not a good thing to do as it can lead to unexpected behaviours and bugs in larger functions where the value coming into the function is no longer what its expected to be, so you should always avoid doing it.
In this case you can do this using the modulus operator, which is similar to division except that instead of returning a decimal when dividing any remainder, it gives a whole number (EG 5 % 3 = 2). This means you dont have to create any variables at all:

```
  return array[index % array.length];
```